src/components/addProductsForm/AddProducts.jsx

```diff
@@ -1,5 +1,181 @@
+    allImages.push(thridImage);
+    const fourthImage = e.target["fourthImage"].value;
+    allImages.push(fourthImage);
+    const allSizes = [];
+    // Getting all sizes and push them into an array
+
+    if (hasSizes === true) {
+      const firstSize = e.target["firstSize"].value;
+      allSizes.push(firstSize);
+      const secondSize = e.target["secondSize"].value;
+      allSizes.push(secondSize);
+      const thirdSize = e.target["thirdSize"].value;
+      allSizes.push(thirdSize);
+      const fourthSize = e.target["fourthSize"].value;
+      allSizes.push(fourthSize);
+    }
+
+    const productInfo = {
+      brand: getProductBrand,
+      title: getProductTitle,
+      categories: getProductCategory,
+      description: getProductDescription,
+      images: allImages,
+      hasSize: hasSizes,
+      sizes: allSizes,
+      price: getProductPrice,
+    };
+
+    await db.collection("products").add(productInfo);
+  };
+
+  const sizeOptions = (
+    <>
+      <option>45</option>
+      <option>44</option>
+      <option>43</option>
+      <option>41</option>
+      <option>40</option>
+    </>
+  );
+
+  return (
+    <>
+      <Form onSubmit={(e) => addNewProductToDatabase(e)}>
+        <Container>
+          <Row>
+            <Col>
+              <Form.Control
+                type="text"
+                placeholder="Brand"
+                onChange={(e) => setGetProductBrand(e.target.value)}
+              />
+            </Col>
+            <Col>
+              <Form.Control
+                type="text"
+                placeholder="Title"
+                onChange={(e) => setGetProductTitle(e.target.value)}
+              />
+            </Col>
+          </Row>
+          <Row>
+            <Col>
+              <Form.Control
+                type="text"
+                placeholder="Description"
+                onChange={(e) => setGetProductDescription(e.target.value)}
+              />
+            </Col>
+          </Row>
+          <Row>
+            <Col xs={12} md={6} lg={5}>
+              <Form.Group controlId="formGridState">
+                <Form.Control
+                  as="select"
+                  defaultValue="Shoes"
+                  onChange={(e) => setGetProductCategory(e.target.value)}
+                >
+                  <option>Shoes</option>
+                  <option>Health care</option>
+                </Form.Control>
+              </Form.Group>
+            </Col>
+            <Col xs={12} md={6} lg={3} className="radioBtn">
+              <Form.Check
+                type="radio"
+                label="Size"
+                name="formHorizontalRadios"
+                id="formHorizontalRadios2"
+                onChange={() => setHasSizes(true)}
+              />
+              <Form.Check
+                type="radio"
+                label="No size"
+                name="formHorizontalRadios"
+                id="formHorizontalRadios1"
```

Comment:
why do these forms have ids?

Reply:
these ids came with react-bootstrap

Reply:
can you show me? i seriously doubt that they did. maybe they're in the example code but that doesn't mean you need them. check if your code breaks if you delete them

Reply:
I deleted the ids and it did not break the code